src/utils.js

```diff
@@ -31,10 +34,10 @@ exports.hasHOCTag = hasHOCTag;
  * @param {Object} member A property definition
  * @returns {Boolean}
  */
-const hasComponentTag = (member) => {
-	// Find any tag field whose `title` is 'required' (won't be there if not required)
+const hasComponentTag = async (member) => {
+	// Find any tag field whose `title` is 'ui' (won't be there if not required)
```

Comment:
Thanks.
But please add this change to "Additional Considerations" in this PR's description.